azkaban-common/src/main/java/azkaban/event/EventData.java

```diff
@@ -0,0 +1,34 @@
+package azkaban.event;
+
+import azkaban.executor.Status;
+
+/**
+ * Carries a snapshot of the status data, suitable for asynchronous message passing.
+ */
+public class EventData {
+
+  private Status status;
+  private String nestedId;
```

Comment:
nestedid seems to be a human readable id for the node. Maybe add a comment that this value corresponds to the return value of azkaban.executor.ExecutableNode#getNestedId?